README.md

```diff
@@ -1,5 +1 @@
-## Welcome to "Hello World" with GitHub Actions
-
-This course will walk you through writing your first action and using it with a workflow file. 
-
-**Ready to get started? Navigate to the first issue.**
\ No newline at end of file
+<! Doctype html>
```

Comment:
<html>

<head>

</head>

<body>
<center><p><h1>Welcome to my Home</h1></p></center><hr>

</body>
</html>